library.h

```diff
@@ -35,6 +35,9 @@
     #define REDIS_VALUE_EXCEPTION(m) zend_value_error(m)
 #endif
 
+#if PHP_VERSION_ID < 80200
+#define zend_mark_function_parameter_as_sensitive(a,b,c)
+#endif
```

Comment:
this conditional part could be removed if https://github.com/php/php-src/pull/8931 is merged (so with 8.2.0beta1)

Reply:
Fixed